ftl/common/builder.py

```diff
@@ -123,8 +123,12 @@ def __init__(self, ctx, namespace, args, cache_version_str,
         self._target_creds = docker_creds.DefaultKeychain.Resolve(
             self._target_image)
         self._transport = transport_pool.Http(httplib2.Http, size=_THREADS)
-        self._base_image = docker_image.FromRegistry(
-            self._base_name, self._base_creds, self._transport)
+        if args.tar_base_image_path:
+            self._base_image = docker_image.FromTarball(
+                args.tar_base_image_path)
+        else:
```

Comment:
maybe make sure that both of these args aren't passed together?

Reply:
I'm not sure I understand what two args you are referring to?  This 'tar_base_image_path' is a value that if set uses a local base image from the path supplied to the flag instead of pulling from a registry

Reply:
sorry, I mean the `tar_base_image_path` and the `base_image`. in theory both of these could get passed, and if they did the `base_image` would get silently ignored, so it might be worth doing a little check to error out if they're both passed since they're mutually exclusive. would just eliminate a little confusion

Reply:
oh I get it now.  def makes sense to have a check for both